tests/Tests/500__jupyterhub/test-jupyterlab-time.robot

```diff
@@ -0,0 +1,84 @@
+*** Settings ***
+Resource            ../../Resources/ODS.robot
+Resource            ../../Resources/Common.robot
+Resource            ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource            ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Library             ../../../../libs/Helpers.py
+Library             DateTime
+Library             OpenShiftCLI
+Library             DebugLibrary
+
+Suite Setup         Load Spawner Page
+Suite Teardown      End Web Test
+
+
+*** Variables ***
+@{LIST_OF_IMAGES}       s2i-minimal-notebook    s2i-generic-data-science-notebook
+...                     pytorch                 tensorflow    minimal-gpu
+
+${LIMIT_TIME} =    40
+
+*** Test Cases ***
+Average Time For Spawning
+    [Documentation]    Verifies that average spawn time for all JupyterHub images is less than 40 seconds
+    [Tags]    ODS-691
+    ...       Tier2
+    ${total_avg} =    Set Variable
+    FOR    ${image}    IN    @{LIST_OF_IMAGES}
+        ${avg} =    Set Variable
+        Spawn and Stop Server    ${image}
+        FOR    ${counter}    IN RANGE    4
+            Close Previous Tabs
```

Comment:
Do you need to do this? Does having too many tabs open create any issues?
You could also close the old browser and open a new one if you wanted to

Reply:
When i added a lines to close the previous tabs then it run successfully

Reply:
is it for closing the JH spanwers?
maybe you could spare the loop and close the window directy in `Spawn and Stop Server`